proposals/0003-nio-postgres.md

```diff
@@ -0,0 +1,280 @@
+# NIOPostgres: A NIO-based PostgreSQL Driver
+
+* Proposal: [SSWG-0003](https://github.com/swift-server/sswg/tree/master/proposals/SSWG-0003.md)
```

Comment:
I've gotten this wrong before - but I think the proper URL (after this is merged) is `sswg/blob/master`